tdms/tests/matlab/data/pstd_input_file.m

```diff
@@ -0,0 +1,141 @@
+%these are not involved in the formal input file spec
```

Comment:
Add a quick comment at the top of this file that explains what it is? Something like "Sample input file used for running unit tests" or similar I imagine.